less/mixins.less

```diff
@@ -1 +1,15 @@
-// Mixins Here
\ No newline at end of file
+// Mixins Here
+.custom-btn() {
+    width: 110px;
+    height: 45px;
+    font-size: 2rem;
+    cursor: pointer;
+    color: @sea-shell;
+    background-color: @btn-bg-color;
+    border-radius: 5px;
+}
+
+// .custom-btn:hover() {
+//     color: @ebony-clay;
+//     background-color: @shuttle-gray;
+// } 
```

Comment:
can you make this a parametric mixin
you just need to add the colour and background-color into the parens () then you can set the colour in the code later on also i think that ':' was throwing the compiler off try switching to a dash or underscore when you are defiinf it in the mixins.less file

.custom-btn-hover(@color,@background-color}{
color: @color;
background-color: @backgorund-color;
}

in the code you would call it
 .custom-btn:hover{
.custom-btn-hover(@ebony-clay,@shuttle-gray}